### PR TITLE
Adds sorting functionality

### LINKED ISSUE #6: Order items

### PR DESCRIPTION
**Pull request by miread**
Fixes some issues that were requested in #15 

Partially implements #6 
Closes #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ Usage: bookmarker [options]
 			Default: Google Chrome Bookmarks
   -d, --description     Set a description for the rendered markdown.
 			Default: Generated by Bookmarker
+  -s, --sort            Set if the list is sorted.
+                        Default: false
```
Comment (lubien) [inline README.md]: Just one last thing.

Can you put this "Default" at the same indent as the other ones?

**Comment by miread** [comment]
Please approve my pull request, and I'll get to work on the next task.

**Comment by lubien** [comment]
Hi @miread I made a review asking for just one fix on your code can you check it out?

**Comment by miread** [comment]
I did take a look at your request.  The problem is that you say the indentation is off - this is a mistake.  The formatting on github is showing an incorrect preview that looks like the indentation is bad.  However, if you click on the README.md file and view the entire contents, you will see that the indentation is all even.  I have double- and triple-checked this.

**Comment by lubien** [comment]
Actually the reason is that you used spaces for indentation while I've used tabs, GitHub caught that.

I've used 3 tabs. You used 24 spaces.

It might result in inconsistencies. I know this is nitpick, do you mind changing it?

**Comment by miread** [comment]
If spacing consistency is that important to you, then you shouldn't be putting tabs into your code.  This issue is your problem, not mine.

**Comment by lubien** [comment]
If you don't want to do this much it's fine, I'll accept the PR then do it myself.

I must say I dislike how you're handling things here blaming design choices of the repo. Please refrain from further contributions.

The code so far you've made is appreciated though. Wish you the best in the future.

**Pull request merged**